br/pkg/storage/s3.go

```diff
@@ -1054,8 +1054,7 @@ func (rs *S3Storage) Create(ctx context.Context, name string, option *WriterOpti
 		}()
 		uploader = s3Writer
 	}
-	uploaderWriter := newBufferedWriter(uploader, WriteBufferSize, NoCompression)
```

Comment:
seems we can not remove this buffer, because the uploaded data will be too small and s3 will reject it.